api-docs/leads.md

```diff
@@ -7,3 +7,85 @@ Here is the list of the valid log types:
 - `leadCreated`,
 - `leadUpdated`,
 - `leadDeleted`
+
+## Get a single Lead
+
+### Request
+
+```http
+GET /api/leads/:id
+```
+
+### Response
+
+```json
+{
+  "id": 1,
+  "dateOfSignup": "2019-09-17",
+  "leadStatus": "active",
+  "contactStage" {
+    "first": "2019-05-06T06:00:00.000Z",
```

Comment:
I like this approach for tracking both status and timestamps all in the same properties. However, one thing that is missing is if the lead is currently active or not. And possibly _why_ the client is inactive -- whether they said they were not interested or if CU just decided not to continue pursuing them. We should maybe create a different property to capture whether they are active or not?